src/test/java/org/terasology/replayTests/WoodCutReplayTest.java

```diff
@@ -33,32 +34,37 @@
         public void run() {
         try {
             String replayTitle = "Woodcut";
-            WoodCutReplayTest.super.openReplay(replayTitle);
+            WoodCutReplayTest.super.openReplay(replayTitle, true);
         } catch (Exception e) {
             e.printStackTrace();
         }
         }
     };
 
-    @Ignore("These are headed tests and should be ignored by Jenkins.")
+    @After
+    public void closeReplay() throws Exception {
+        super.getHost().shutdown();
+        GameThread.reset();
+        replayThread.join();
+    }
+
     @Test
     public void testWoodcut() {
         replayThread.start();
         try {
```

Comment:
Same as above -- catching exceptions in a test (can't comment on the catch block). Big no-no. If you've caught exceptions in tests elsewhere, I'd recommend you get rid of those `try...catch` blocks too.